packages/typespec-test/test/batch_modular/generated/typespec-ts/review/batch.api.md

```diff
@@ -1645,6 +1645,20 @@ export interface OutputFileUploadOptions {
     uploadCondition: OutputFileUploadCondition;
 }
 
+// @public
+export interface PagedAsyncIterableIterator<TElement, TPage = TElement[], TPageSettings = PageSettings> {
+    [Symbol.asyncIterator](): PagedAsyncIterableIterator<TElement>;
+    byPage: (settings?: TPageSettings) => AsyncIterableIterator<TPage & {
+        continuationToken?: string;
+    }>;
```

Comment:
I gave up to name this return type with a specific name like `ContinuablePage` because I found directly using `TPage & { continuationToken?: string; }` could be easier to discover the real type e.g User[] & { continuationToken?: string; } and they don't need to click into the definition.

```ts
export type ContinuablePage<TPage> = TPage & { continuationToken?: string };
```